src/components/List/List.scss

```diff
@@ -26,4 +29,49 @@
             }
         }
     }
+}
+
+.list.pink {
+    .list__item {
+
+        &:hover {
+            color: #F65261;
+
+            &::after {
+                background-color: #F65261;
+            }
+        }
+    }
+
```

Comment:
please keep same spaces

Reply:
redundant space removed, thank you for your comment.